README.md

```diff
@@ -29,30 +29,31 @@ Changes might occur which impact applications that use this SDK.
 
 <!-- toc -->
 
-- [Overview](#overview)
-- [Features](#features)
-- [Prerequisites](#prerequisites)
-- [Installation](#installation)
-  * [Maven](#maven)
-  * [Gradle](#gradle)
-- [Getting started](#getting-started)
-  * [Authentication](#authentication)
-    + [IAM authentication](#iam-authentication)
-    + [Session cookie authentication](#session-cookie-authentication)
-    + [Basic authentication](#basic-authentication)
-  * [Code examples](#code-examples)
-    + [1. Retrieve information from an existing database](#1-retrieve-information-from-an-existing-database)
-    + [2. Create your own database and add a document](#2-create-your-own-database-and-add-a-document)
-    + [3. Update your previously created document](#3-update-your-previously-created-document)
-    + [4. Delete your previously created document](#4-delete-your-previously-created-document)
-- [Error handling](#error-handling)
-- [Using the SDK](#using-the-sdk)
-- [Questions](#questions)
-- [Issues](#issues)
-- [Further resources](#further-resources)
-- [Open source @ IBM](#open-source--ibm)
-- [Contributing](#contributing)
-- [License](#license)
+- [IBM Cloudant Java SDK Version 0.0.18](#ibm-cloudant-java-sdk-version-0018)
```

Comment:
Did the TOC make this?

Reply:
I think the IDE plugin must have added that.

Reply:
Can I ask you to rebase this part? It will ease my ongoing PR, plus I do not see much benefit in having the title as part of the table of contents.

Reply:
Removed in 5afe3c4e26007afa4b208ea79998da678ca2e251